app/src/main/java/com/example/concordia_campus_guide/Models/Coordinates.java

```diff
@@ -47,4 +48,8 @@ public double getLongitude() {
     public void setLongitude(double longitude) {
         this.longitude = longitude;
     }
+
+    public LatLng getLatLng(){
+        // this is not a bug
+        return new LatLng(this.longitude, this.latitude);}
```

Comment:
can we not do that? we should investigate this and fix the root cause, as we are not using the LatLng object properly

Reply:
The lat and long of the walking points for VL were indeed inverted but it has been fixed now [(PR165).](https://github.com/Concordia-Campus-Guide/Concordia-Campus-Guide/pull/178/commits/6c69aa32a525bbaf4945065d37514face550640f) Just letting you know in case this was a temp fix for that.